access-control-app/project.clj

```diff
@@ -59,4 +73,12 @@
                                         :let [project-dir (.replace (name group-artifact) "cmr-" "")]]
                                     ["shell" "ln" "-s" (str "../../" project-dir) "checkouts/,"]))
             ;; Alias to test2junit for consistency with lein-test-out.
-            "test-out" ["test2junit"]})
+            "test-out" ["test2junit"]
+            ;; Linting aliases
+            "kibit" ["do" ["with-profile" "lint" "shell" "echo" "== Kibit =="]
+                          ["with-profile" "lint" "kibit"]]
+            "eastwood" ["with-profile" "lint" "eastwood" "{:namespaces [:source-paths]}"]
+            "bikeshed" ["with-profile" "lint" "bikeshed"]
```

Comment:
To keep things clean, the plugin was added to a `lint` profile ... the only way that bikeshed will get run is by executing `lein with-profile lint bikeshed`, and thus the alias. If bikeshed had been added to the top-level of each project, and not in a profile, then exactly what you said would apply.